src/packages/utils.ts

```diff
@@ -85,5 +86,8 @@ export const exportToSvg = ({
     ...appState,
     exportPadding,
     metadata,
+    exportBackground: appState.exportBackground ?? true,
+    viewBackgroundColor: appState.viewBackgroundColor ?? oc.white,
+    shouldAddWatermark: appState.shouldAddWatermark ?? false,
```

Comment:
why do we need this?

Reply:
So that if these params are not passed we use default values same like we do `exportAsCanvas`. But i think spreading `defaultAppState` would be better in both ?

Reply:
I see. Then we need to change the type of `ExportOpts.appState` to `Partial<Omit<AppState, "offsetTop" | "offsetLeft">>`, and run `restoreAppState(appState)` instead of doing it manually. For that matter we should restore even `ExportOpts.elements`.